src/content/ui/adaptive-responsive/more-info.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Additional resources
+description: >-
+  Other resources that you might find useful when writing adaptive apps.
+short-title: Learn
+---
+
+## Examples
+
+If you'd like to see how these concepts come together,
```

Comment:
Consider being more specific about what "these concepts" refers to as readers may navigate to this page from elsewhere.